src/ontology/mondo-ingest.Makefile

```diff
@@ -423,8 +441,7 @@ matches: lexical-matches extract-unmapped-matches
 # Related issues:
 #  - icd10cm/icd10who ttl -> owl: https://github.com/monarch-initiative/mondo-ingest/issues/138
 #  - No rule to make target 'mirror/ONTOLOGY.owl': https://github.com/monarch-initiative/mondo-ingest/issues/137
-# Maybe solved now by using $(COMPONENTSDIR)/
-$(COMPONENTSDIR)/%.db: $(COMPONENTSDIR)/%.owl
+$(COMPONENTSDIR)/%.db:
```

Comment:
If you dont add the dependency, goals depending on the DB file will end up working on an outdated version of the component in some cases

Reply:
Hmmmm... I'm really grateful for your reviews. I agree with you. I don't remember this being intentional. This is something I did temporarily because of the [ODK issue](https://github.com/INCATools/ontology-development-kit/pull/880#event-9424556509). I added it back to `%-db` but forgot to add it here. It looks like the comments above are no longer an issue for crating a `.db` so I just removed them.